string type
string name
string path
---
int32 status
string message